js/main.js

```diff
@@ -143,6 +143,59 @@ $('body').on('keypress', 'input', function(event) {
   }
 });
 
+//keyboard shortucts
+window.keyShortcuts = {
+	discover: 		'd',
+	myPage: 		'h',
+	customizePage: 	'e',
+	create: 		'n',
+	purchases: 		't',
+	sales: 			's',
+	cases: 			'b',
+	settings: 		','
```

Comment:
I'm a Mac user, I didn't know that. It must be the fact I only use my Mac sometimes, I've been tainted by dark forces. ;)

I think so far the majority of our users are on Windows, using patterns familiar to Mac users isn't going to be a benefit for most people.

Any random letter is probably fine, maybe "e"?